securedrop/journalist_app/forms.py

```diff
@@ -58,7 +58,6 @@ class ReplyForm(FlaskForm):
 class LogoForm(FlaskForm):
     logo = FileField(validators=[
         FileRequired(message=gettext('File required.')),
-        FileAllowed(['jpg', 'png', 'jpeg'],
-                    message=gettext("You can only upload JPG/JPEG"
-                                    " or PNG image files."))
+        FileAllowed(['png'],
+                    message=gettext("You can only upload PNG image files."))
```

Comment:
If we change that to **Upload images only.** it will alow us to re-use a former translation instead of asking them for a last minute change.

Reply:
I think that's fair since this is a last minute change. I've modified this to reuse this translation ✅.

Once this PR is merged, I'll prepare a PR (_not_ for 0.7.0 :innocent:) changing the string to "You can only upload PNG image files." since that's more clear for end users.